services/simcore/docker-compose.yml

```diff
@@ -1,9 +1,44 @@
-# osparc-simcore stack (framework stack)
-# NOTES :
-# LOGSPOUT_MULTILINE is used to tell to Logspout, which is used by GrayLog, to handle multiline support of the logs of the concerned container. Please add with every new osparc-simcore service.
-# See this issue to have more informations : https://github.com/ITISFoundation/osparc-ops/issues/40
 version: "3.8"
 services:
+  clusters-keeper:
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}"
```

Comment:
in osparc-simcore we define it as: ```hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"```
The task.Slot copes with the fact that in case of scaling up the service and more than 1 replica ends up on the same machine that they still have different names.
This only does not work in case of global service. I would propose to just remove that line.

Reply:
thats correct I guess, thanks for catching it, will remove the line